config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+end
```

Comment:
- [ ] Let's create API endpoints for our app, when this file is empty, then we can't have access to the data in our database. Endpoints are like addresses to the data we want to have access to.